proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -619,20 +629,65 @@ def eq_flux_io(b, t, x, p, j):
                          self.config.property_package.component_list,
                          doc="Mass transfer from feed to permeate")
         def eq_connect_mass_transfer(b, t, p, j):
-            return (b.properties_permeate[t].get_material_flow_terms(p, j)
+            return (b.permeate_side.properties_permeate[t].get_material_flow_terms(p, j)
                     == -b.feed_side.mass_transfer_term[t, p, j])
 
         @self.Constraint(self.flowsheet().config.time,
                          doc="Enthalpy transfer from feed to permeate")
         def eq_connect_enthalpy_transfer(b, t):
-            return (b.properties_permeate[t].get_enthalpy_flow_terms('Liq')
+            return (b.permeate_side.properties_permeate[t].get_enthalpy_flow_terms('Liq')
                     == -b.feed_side.enthalpy_transfer[t])
 
         @self.Constraint(self.flowsheet().config.time,
                          doc="Isothermal assumption for permeate")
         def eq_permeate_isothermal(b, t):
             return b.feed_side.properties_out[t].temperature == \
-                   b.properties_permeate[t].temperature
+                   b.permeate_side.properties_permeate[t].temperature
+
+        # # Permeate-side instantaneous stateblocks
+        @self.permeate_side.Constraint(self.flowsheet().config.time,
+                                   self.io_list,
+                                   self.solute_list,
+                                   doc="Permeate instantaneous mass fraction")
```

Comment:
To be consistent with other inlet/outlet variables, I'd remove the word "instantaneous" in all places, including the "instant" qualifier.
```suggestion
                                   doc="Permeate mass fraction at inlet and outlet")
```